config/deploy.rb

```diff
@@ -40,7 +40,7 @@
 # Uncomment the following to require manually verifying the host key before first deploy.
 # set :ssh_options, verify_host_key: :secure
 
-set :whenever_roles, [:app]
+set :whenever_roles, [:cron]
```

Comment:
The intent here is to prevent whenever from setting up any cronjobs on our servers, none of which use the `:cron` Capistrano role, I believe. I wonder if you are open to doing the following:

```suggestion
# NOTE: Commented out to prevent crontab updates. Default whenever roles are `[:db]` which we do not use for rialto-etl.
# set :whenever_roles [:app]
```